### PR TITLE
Eq 412 2016 census schema

### PR DESCRIPTION
**Pull request by weapdiv-david**
### What is the context of this PR?

This PR adds the Census 2016 Household Test questions
### How to review

1) Checkout the branch
2) Run the unit tests
3) Run the app and go to the dev console
4) Select `0_hh2016.json` from the list of surveys
5) Fill in the questionnaire
- [ ] Do all commits have sensible commit messages?
- [ ] Is all new code unit tested?
- [ ] Are there integration tests if needed?
- [ ] Is there sufficient logging?
- [ ] Is there documentation or is the code self-describing?
### Who worked on the PR

@weapdiv-david worked on this, so it will need to be reviewed by someone else

### CODE REVIEW

File: app/data/0_hh2016.json [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+                                                "messages": {}
+                                            }
+                                        }
+                                    ],
+                                    "description": "Please choose <strong>only one</strong> of the following:",
+                                    "display": {
+                                        "properties": {}
+                                    },
+                                    "id": "pef6r645-c231-4522-ba2f-273b25507e53",
+                                    "title": "Did you visit the help facility at&nbsp;<a href=\"http://www.ons.gov.uk/census\">www.ons.gov.uk/census</a>&nbsp;?",
```
Comment (dhilton) [inline app/data/0_hh2016.json]: This link gets escaped and quoted; as we can't support links yet can we just use the web address for now?

**Comment by dhilton** [comment]
LGTM apart from link issue mentioned above but need to get @ajmaddaford & co to check.

**Comment by weapdiv-david** [comment]
Updated the schema to remove the link...

@ajmaddaford has a copy of the schema (with link) as well as a copy of the data submission generated from this by eQ.

Awaiting feedback

**Comment by dhilton** [comment]
LGTM 👍

**Pull request merged**